### ISSUE TITLE
Get all bookings for an amenity with an optional time range

### ISSUE THREAD

**Issue by jmprathab** [jmprathab/MyHome#114]
## ⚠️ Is your feature request related to a problem? Please describe

* Get all bookings for an amenity with an optional time range

## 💡 Describe the solution you'd like

* `GET /amenities/{amenityId}/bookings?start='01-01-2020'&end='01-02-2020'`
* The API should support pagination
* Code pointers for pagination - `CommunityService.java::listAll`
* Update postman collection

## Task Relationship

Subtask of: jmprathab/MyHome#94

**Comment by DDmit04** [comment]
Is pagination needed for result bookings list?

**Comment by jmprathab** [comment]
> Is pagination needed for result bookings list?

Yes. It is required. I will modify the comments to include that. Thanks!

**Comment by nlaert** [comment]
Hello, I will take this ticket for starting, is that ok? :)

**Comment by jmprathab** [comment]
@nlaert Sure. I have assigned it to you.

**Comment by bayotheman** [comment]
Hi @jmprathab @mslowiak , can I take on this task?

**Comment by bayotheman** [comment]
hi @mslowiak  I have made the PR to this issue
